package.json

```diff
@@ -214,9 +214,11 @@
         "fsevents": "^2.1.3"
     },
     "dependencies": {
+        "@microsoft/applicationinsights-core-js": "^2.8.14",
```

Comment:
@Vidya2606 ```applicationinsights-core-js``` and ```tslib``` are required dependencies for new vscode telemetry ? Just wondering what are these dependencies used for ?

Reply:
Yes, these are transitive dependencies. Since the node version was an older one was getting this error. I have updated the node version, this should take care of this.